third_party/docfx/templates/devsite/UniversalReference.common.js

```diff
@@ -64,6 +64,9 @@ exports.transform = function (model) {
     }
   }
 
+  if (model.summary != null) {

```

Comment:
```suggestion
  if (model.summary != null && model.friendlyApiName) {
```

This is going to run for all partials that don't include `friendlyApiName`, let's exclude that rather than checking for it within the function afterwards.